src/main/kotlin/lotto/io/InputView.kt

```diff
@@ -12,6 +13,19 @@ object InputView {
         return requireNotNull(readlnOrNull()?.toIntOrNull()) { INPUT_ERROR_MESSAGE }
     }
 
+    fun getManualLottoSize(): Int {
+        println(MANUAL_LOTTO_SIZE_INPUT_MESSAGE)
+        return requireNotNull(readlnOrNull()?.toIntOrNull()) { INPUT_ERROR_MESSAGE }
+    }
+
+    fun getManualLotto(): List<Int> {
+        return requireNotNull(
+            readlnOrNull()?.split(INPUT_DELIMITER)?.map {
+                requireNotNull(it.toIntOrNull()) { INPUT_ERROR_MESSAGE }
+            }
+        ) { INPUT_ERROR_MESSAGE }
+    }
```

Comment:
이렇게 `null safe` 연산이나 함수를 중첩해서 사용하면 가독성이 떨어질 수 있어요.
이런 경우 null check를 함수 최상단에서 수행한 후 non-null 타입으로 스마트 캐스팅 시키면 조금 더 가독성있는 코드가 될 수 있습니다!
```suggestion
    fun getManualLotto(): List<Int> {
        val input = readlnOrNull()
        require(!input.isNullOrBlank()) { INPUT_ERROR_MESSAGE }

        return input.split(INPUT_DELIMITER)
            .map { it.toIntOrNull() ?: throw IllegalArgumentException(INPUT_ERROR_MESSAGE) }
    }
```

Reply:
우와 작성할땐 몰랐는데 지금 보니까 제 코드 보기 어렵네요 ㅋㅋㅋㅋ
감사합니다 👍 
쪼금더 단축해서 아래처럼 해봤는데 어떤가요?
```
        val input = requireNotNull(readlnOrNull()) { INPUT_ERROR_MESSAGE }
        return input.split(INPUT_DELIMITER)
            .map { it.toIntOrNull() ?: throw IllegalArgumentException(INPUT_ERROR_MESSAGE) }
```